### PR TITLE
Add space pals to the project list and fix a typo in the docs

### PR DESCRIPTION
**Pull request by januarionclx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Don't forget to give the project a star! Thanks again!
 ### Running tests locally
 
 1. `npm install`
-2. `npm node`
+2. `npm run node`
```
Comment (cygaar) [inline README.md:86]: Can you remove 3, and change 2 to `npm run test`

**Review by cygaar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (januarionclx) [reply README.md:86]: You don't want npm run node anymore? regarding npm run test, I could, but npm test is a known command, npm run node is not, LMK if you want me to edit it anyways.

Comment (cygaar) [reply README.md:86]: You don't need npm run node to run the unit tests

**Comment by cygaar** [comment]
Someone else just opened a PR for that, so feel free to rebase onto main

**Pull request closed without merging**